README.md

```diff
@@ -43,7 +43,8 @@ java -jar swagger-codegen-cli-2.4.5.jar generate \
  1. Check the generated files and copy them into appropriate source directory. Do not overwrite existing files.
  1. Remove Links, Meta, OBError1 and OBErrorResponse1 - we use shared generic versions of these classes.
  1. Repeat generation for each new swagger json file
- 1. If using Intelij, run format and optimise imports on newly generated files. 
+ 1. If using Intelij, run format and optimise imports on newly generated files.
+ 1. Fix imports where necessary - e.g. use the Links and Meta from the `account` package and use `org.joda.time.DateTime` instead of `org.threeten.bp.OffsetDateTime.OffsetDateTime`.
```

Comment:
I would have preferred to use Java 8 dates, but Joda is used in all of the other generated classes in this repo.